rasa_nlu_examples/tokenizers/stanzatokenizer.py

```diff
@@ -0,0 +1,128 @@
+from typing import Any, Dict, List, Text
+
+from rasa.nlu.tokenizers.tokenizer import Token, Tokenizer
+from rasa.nlu.training_data import Message
+from functools import reduce
+from rasa.nlu.tokenizers.spacy_tokenizer import POS_TAG_KEY
+
+import stanza
+
+
+class StanzaTokenizer(Tokenizer):
+    """
+    The StanzaTokenizer allows for more pos/lemma features to be used in the
+    Rasa ML pipelines. It is based on the project found here: https://stanfordnlp.github.io/stanza/usage.html
+
+    Not every language here has good performance metrics. To see the details
+    check out this table: https://stanfordnlp.github.io/stanza/performance.html
+
+    Before running the stanza model in production, be sure to check the license information
+    since it may differ per language: https://stanfordnlp.github.io/stanza/available_models.html
+    """
+
+    defaults = {
+        # What language to use
+        "lang": None,
+        # What package to use, details on this can be found here:
+        # https://stanfordnlp.github.io/stanza/download_models.html
+        "pkg": "package",
```

Comment:
you are not using this parameter

Reply:
It's an advanced feature of the library that I might add in the future. For now though, seems fine to turn that into an issue and remove it.